test/modules/gs1-importer.test.js

```diff
@@ -37,6 +38,28 @@ describe('GS1 Importer tests', () => {
         { args: [path.join(__dirname, '../../importers/xml_examples/GraphExample_2.xml')] },
         { args: [path.join(__dirname, '../../importers/xml_examples/GraphExample_3.xml')] },
         { args: [path.join(__dirname, '../../importers/xml_examples/GraphExample_4.xml')] },
+        { args: [path.join(__dirname, '../../importers/xml_examples/Basic/01_Green_to_pink_shipment.xml')] },
```

Comment:
We should have unique file names, so for example 01_Green_to_pink_shipment.xml in varous subfolders should not be called the same
@vukdjoric